rele/client.py

```diff
@@ -40,11 +40,8 @@ class Subscriber:
     def __init__(self, gc_project_id, credentials, default_ack_deadline=None):
         self._gc_project_id = gc_project_id
         self._ack_deadline = default_ack_deadline or DEFAULT_ACK_DEADLINE
-
-        if USE_EMULATOR:
-            self._client = pubsub_v1.SubscriberClient()
-        else:
-            self._client = pubsub_v1.SubscriberClient(credentials=credentials)
+        self.credentials = credentials if not USE_EMULATOR else None
+        self._subscriber_client = pubsub_v1.SubscriberClient(credentials=credentials)
```

Comment:
I still see no reason to rename `Subscriber._client` to `Subscriber._subscriber_client`. `client` profits from the class namespace. Plus now this class will have inconsistent naming compared to `Publisher`

Reply:
I disagree but I have already changed it. 👍